python-checks/src/test/resources/checks/useOfAnyAsTypeHintImported.py

```diff
@@ -0,0 +1,3 @@
+from abc import ABCMeta
+
+class ImportedParentWithMetaClass(metaclass=ABCMeta): ...
```

Comment:
_minor_ I suggest adding a class that has no metaclass in this file. Then, you can add a child class in the other file to confirm that the metaclass is really the root cause.